include/libdnf/rpm/solv_query.hpp

```diff
@@ -358,10 +358,12 @@ class SolvQuery {
     /// Return the number of packages in the SolvQuery.
     ///
     /// @replaces libdnf/sack/query.hpp:method:Query.size()
-    std::size_t size() const noexcept;
+    // inherited from PackageSet
+    // std::size_t size() const noexcept;
```

Comment:
Are you sure that the dock string will work correctly?

Reply:
I will remove the lines.